src/emmodel/model.py

```diff
@@ -126,4 +126,75 @@ def plot_time_trend(ax: plt.Axes, df: pd.DataFrame,
     ax.set_xlabel("time")
     for time in year_heads:
         ax.axvline(time, linestyle="--", color="gray")
-    return ax
\ No newline at end of file
+    return ax
+
+
+class LinearExcessMortalityModel:
+    """LinearExcessMortalityModel"""
+    def __init__(self, df: pd.DataFrame, col_obs: str, 
+                 stage1_col_covs: List[str], stage2_col_covs: List[str],
+                 col_covs_priors: List[str], model_type: str = 'two-stage'):
+        super(LinearExcessMortalityModel, self).__init__()
+        self.df = df
+        self.col_obs = col_obs
+        self.stage1_col_covs = stage1_col_covs
+        self.stage2_col_covs = stage2_col_covs
+        self.model_type = model_type
+        self.col_covs_priors = col_covs_priors
+
+    def run(self, prior_min_var: float = 0.1):
+        if self.model_type == 'two-stage':
```

Comment:
are there other types?

Reply:
I was thinking of it as a placeholder, since in the previous model, they wanted only one stage for certain locations. If not, we could certainly drop the argument.